README.md

```diff
@@ -1,5 +1,5 @@
 # baseplate.go
-Baseplate implemented in go
+[Baseplate](https://github.com/reddit/baseplate.py) implemented in go
```

Comment:
I don't think this is the correct change, although it indeed provides more context than before.